dartdoc_test/.gitignore

```diff
@@ -0,0 +1 @@
+.dartdoc_test
```

Comment:
Do we still create this, or do we create things inside `.dart_tool/dartdoc_test/`